packages/popover/stories/popover.stories.ts

```diff
@@ -8,19 +8,54 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
-import { html, boolean, text, radios } from '@open-wc/demoing-storybook';
-
 import '../sp-popover.js';
-import { TemplateResult } from '@spectrum-web-components/base';
+import { html, TemplateResult } from '@spectrum-web-components/base';
+import { Placement } from '@spectrum-web-components/overlay/src/popper';
```

Comment:
Is `Placement` available in the main entry point `'spectrum-web-components/overlay'`?